src/services/ubi/community.ts

```diff
@@ -1582,4 +1548,156 @@ export default class CommunityService {
             metrics: metrics !== null ? [metrics] : undefined,
         };
     }
+
+    private static _generateInclude(fields: any): Includeable[] {
+        const extendedInclude: Includeable[] = [];
+        if (fields.suspect) {
+            extendedInclude.push({
+                model: this.ubiCommunitySuspect,
+                attributes:
+                    fields.suspect.length > 0 ? fields.suspect : undefined,
+                as: 'suspect',
+                required: false,
+                duplicating: false,
+                where: {
+                    createdAt: {
+                        [Op.eq]: literal(
+                            '(select max("createdAt") from ubi_community_suspect ucs where ucs."communityId"="Community".id and date("createdAt") > (current_date - INTERVAL \'1 day\'))'
+                        ),
+                    },
+                },
+            });
+        }
+
+        if (fields.cover || fields.thumbnails) {
+            extendedInclude.push({
+                attributes: fields.cover
+                    ? fields.cover.length > 0
+                        ? fields.cover
+                        : undefined
+                    : ['id'],
+                model: this.appMediaContent,
+                as: 'cover',
+                duplicating: false,
+                include: fields.thumbnails
+                    ? [
+                          {
+                              attributes:
+                                  fields.thumbnails.length > 0
+                                      ? fields.thumbnails
+                                      : undefined,
+                              model: this.appMediaThumbnail,
+                              as: 'thumbnails',
+                              separate: true,
+                          },
+                      ]
+                    : [],
+            });
```

Comment:
Thinking of how the frontend will use it, I don't see any advantage on this. Every parameter is necessary and usually, from the array of thumbnails, only one of them (width some chosen size) is used. So I guess, we can just return all values and not use the `fields.thumbnails` as a condition.
I think we can return only a given size, but lets leave that for later.

Reply:
got it!
I did the change to return all fields of thumbnails